fluxx.game.php

```diff
@@ -255,8 +268,7 @@ public function drawExtraCards($player_id, $drawCount)
       "cards" => $cardsDrawn,
     ]);
 
-    self::notifyAllPlayers(
-      "cardsDrawnOther",
+    self::notifyAllPlayers("cardsDrawnOther",

```

Comment:
@tomlowi : we should probably agree on a linter for the php files. 
I'm using prettier with the php plugin but can change to whatever suits you best.